packages/ethereum-storage/test/smartcontract-manager.test.ts

```diff
@@ -252,15 +252,17 @@ describe('SmartContractManager', () => {
 
     // This mock is used to ensure any block is never fetchable
     smartContractManager.eth.getBlock = (_block: any): any => {
-      return null;
+      throw 'some error';
     };
 
-    await expect(
-      smartContractManager.addHashAndSizeToEthereum(hashStr, { contentSize: otherSize }),
-    ).rejects.toThrowError('Maximum number of confirmation reached');
-
-    clearInterval(blockInterval);
-  }, 30000);
+    try {
+      await expect(
+        smartContractManager.addHashAndSizeToEthereum(hashStr, { contentSize: otherSize }),
+      ).rejects.toThrowError('Maximum number of confirmation reached');
+    } finally {
+      clearInterval(blockInterval);
```

Comment:
This is to clarify test failures. When this test failed, it tended to make other tests of the package fail as well.